docs/pages/example/timeline-animation.html

```diff
@@ -54,7 +54,7 @@ <h2>Significant earthquakes in 2015</h2>
     </div>
 </div>
 
-<script src="//d3js.org/d3.v3.min.js" charset="utf-8"></script>
+<script src="https://d3js.org/d3.v6.min.js" charset="utf-8"></script>
```

Comment:
@HeyStenson Maybe try v3? It's possible v6 has breaking changes.

```suggestion
<script src="https://d3js.org/d3.v3.min.js" charset="utf-8"></script>
```